File: golang/src/entidades/db/pessoa.go
package db

type Pessoa struct {
	Id     uint    `gorm:"column:id;primaryKey" json:"id" form:"id" `
	Nome   string  `gorm:"column:nome" json:"nome" form:"nome"`
	Carros []Carro `gorm:"foreignKey:DonoId" json:"carros"`
}

func (Pessoa) TableName() string {
	return "pessoa"
}

File: nodejs/src/entidades/db/pessoa.ts
import { DataTypes } from 'sequelize';
import { db } from '../../config/db';

export const Pessoa = db.define('Pessoa', {
  id: {
    primaryKey: true,
    type: DataTypes.INTEGER,
    autoIncrement: true
  },
  nome: {
    type: DataTypes.STRING(255)
  }
}, {
  tableName: 'pessoa',
  timestamps: false
});
